tests/src/case0038.c

```diff
@@ -32,8 +32,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#ifdef HAVE_ORACLE
```

Comment:
This must be removed because the file is not compiled if Oracle is not configured and HAVE_ORACLE is not passed in tests.

Reply:
I am sure I needed to add this as the test was being compiled despite not having Oracle configured. I do not think the test was run, only compiled. I will investigate the "check" dependencies - these are complex hence my change as the easiest way to avoid the problem. But if HAVE_ORACLE is not passed to test builds...